src/language/grammar.tx

```diff
@@ -1,12 +1,18 @@
 Specification:
+	params = Params?
 	actors += Actor*
-	( 'maneuverJunction' '(' maneuverJunction = ID ')' )?
 	constraints += Constraint*
 ;
 
-Actor: Car | Pedestrian
+Params:
+	'Params' '{'
+		( 'map' ':' map = STRING )?
+		( 'maneuverJunction' ':' maneuverJunction = ID )?
+	'}'
```

Comment:
How does this kind of syntax look like? Maybe we can do something similar (section, brackets explicitly) for actors and constraints as well?

One somewhat weird thing though is that the order of parameters will be fixed (we can't have maneuverJunction before map).